attributes/default.rb

```diff
@@ -1,30 +1,5 @@
-default['stackstorm']['api_url'] = 'http://127.0.0.1:9101'
-default['stackstorm']['home'] = '/opt/stackstorm'
-default['stackstorm']['bin_dir'] = '/usr/bin'
-default['stackstorm']['etc_dir'] = '/etc/st2'
-default['stackstorm']['conf_path'] = '/etc/st2/st2.conf'
-default['stackstorm']['log_dir'] = '/var/log/st2'
-default['stackstorm']['run_user'] = 'st2'
-default['stackstorm']['run_group'] = 'st2'
+# List of available `st2` services:
+# https://github.com/StackStorm/st2/blob/master/st2common/bin/st2ctl#L5
+default['stackstorm']['SERVICES'] = %w(st2actionrunner st2api st2stream st2auth st2garbagecollector st2notifier st2resultstracker st2rulesengine st2sensorcontainer).freeze
```

Comment:
I would look at using a library helper to return if it you need it in multiple places
Otherwise i would define a local variable in the recipe that its needed in